CONTRIBUTING.md

```diff
@@ -128,7 +128,7 @@ image, if you need to have another binary available feel free to make a PR to th
 
 https://github.com/tektoncd/plumbing/blob/master/tekton/images/test-runner/Dockerfile
 
-An helper script called [run-test.sh](test/run-test.sh) is provider in the
+A helper script called [`run-test.sh`](test/run-test.sh) is provider in the
```

Comment:
maybe add the requirements deps in the contributing documentation here?

Reply:
I thought about it, hence I was looking at it, but IME it would prove difficult to try to keep both areas in sync. My train of thought was that it would be more beneficial and easier to maintain if it was in the script where they are used.